remix.init/README-edge.md

```diff
@@ -39,6 +39,8 @@ npm run dev
 
 Open up [http://localhost:3000](http://localhost:3000), and you should be ready to go!
```

Comment:
I've updated the Netlify Edge Functions and Netlify READMEs to be exactly the same aside from the link to Netlify Functions or Edge Functions docs. That should be the last changes required @stephmarie17 unless you spot anything else. Thanks again!